benchmark/single-source/SetTests.swift

```diff
@@ -55,6 +55,36 @@ let setQ: Set<Int> = {
   return set
 }()
 
+// Construction of empty array.
```

Comment:
I made arrays as sequences.
Are they ok?

Reply:
I have initially thought you could have gone directly with ranges, which conform to `Sequence`, but for all the `Box` variants it is better to materialize the objects into arrays, otherwise the benchmark would have been dominated by reference counting traffic… so I think using arrays like you did is OK.